Examples/arks_test-demo/README.txt

```diff
@@ -0,0 +1,11 @@
+#LEC2018
+## ARKS demo scripts
+
+This test is provided for the sole purpose of testing your ARCS installation.
+
+Once you have all the software in your path, execute ./runARCSarks-demo.sh 
+
+Output files generated from an in-house run with 
+ARCS v1.0.2
```

Comment:
Hmm this isn't really correct to say it is ARCS v1.0.2... I know why you changed it, but let's just change to what commit was used for now (or the up-to-date commit), and I can update when we do the release. Have you double checked that the provided outputs still match the output of `ARCS --arks`?

Reply:
I changed it to ARCS v 1.0.6 and yes the outputs match.

Reply:
But unfortunately this won't work with ARCS v1.0.6, since that doesn't have the ARKS integration. I'll change to v1.0.7 when we are ready to do a release, but let's change to a commit ID or say 'current master' or the like for now.

Reply:
Hmm OK.